src/main/java/com/google/api/codegen/discogapic/transformer/java/JavaDiscoGapicRequestToViewTransformer.java

```diff
@@ -245,21 +241,51 @@ private StaticLangApiMessageView generateRequestClass(
     }
 
     for (FieldModel entry : method.getInputFields()) {
+      if (entry.mayBeInResourceName()) {
+        hasRequiredProperties = true;
+        continue;
+      }
       String parameterName = entry.getNameAsParameter();
       properties.add(
           schemaToParamView(context, entry, parameterName, symbolTable, EscapeName.ESCAPE_NAME));
       if (entry.isRequired()) {
         hasRequiredProperties = true;
       }
     }
+
+    StaticLangApiMessageView.Builder paramView = StaticLangApiMessageView.newBuilder();
+    Method discoMethod = ((DiscoveryMethodModel) method).getDiscoMethod();
+    String resourceName = DiscoGapicNamer.getResourceIdentifier(discoMethod.path()).toLowerCamel();
+    StringBuilder description =
+        new StringBuilder(discoMethod.parameters().get(resourceName).description());
+    description.append(String.format("\nIt must have the format `%s`. ", discoMethod.path()));
+    description.append(String.format("\\`{%s}\\` must start with a letter,\n", resourceName));
+    description.append(
+        "and contain only letters (\\`[A-Za-z]\\`), numbers (\\`[0-9]\\`), dashes (\\`-\\`),\n"
+            + "     * underscores (\\`_\\`), periods (\\`.\\`), tildes (\\`~\\`), plus (\\`+\\`) or percent\n"
+            + "     * signs (\\`%\\`). It must be between 3 and 255 characters in length, and it\n"
+            + "     * must not start with \\`\"goog\"\\`.");
```

Comment:
I don't see this showing up in the baseline, did it not get piped through correctly?

Reply:
The description comments aren't yet rendered by the snippets.